scripts/west_commands/runners/stm32flash.py

```diff
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: Apache-2.0
+
+'''Runner for flashing with stm32flash.'''
+
+import os
+import platform
+import sys
+
+from runners.core import ZephyrBinaryRunner, RunnerCaps
+
+default_device = '/dev/ttyUSB0'
+if platform.system() == 'Darwin':
+    default_device = '/dev/tty.SLAB_USBtoUART'
+    
+class Stm32flashBinaryRunner(ZephyrBinaryRunner):
+    '''Runner front-end for stm32flash.'''
+
+    def __init__(self, cfg, device, start_address='0x0', 
+                 baud='57600', firmware=None):
```

Comment:
The firmware argument is ignored. Should it be removed? Or should there be something like `self.firmware = firmware or cfg.bin_file`?